apify/apify-zapier-integration
# feat: add nice UI for actor run input generated from input schema

# Updates

Major updates:
* Parsing input schema and generate UI in Zapier in run Actor action
* For Actor without schema there will be input body and input content encoding as it is now instead of generated UI

Minor updates
* remove important field it was deprecated in https://github.com/zapier/zapier-platform/pull/644
* improve list of tasks / actors dropdown
* fix Actor name
* update zapier-core package
* update node version
* ...

# Testing in Zapier
* You need to be in Apify Zapier developer group or use info@apify.com user from 1pass to login
* Create new Zap and select the trigger, the easiest way is to use "Catch Hook" or "Actor run finished"
![image](https://github.com/apify/apify-zapier-integration/assets/11837643/a59fdb54-0a3c-4be8-ac95-43e878a981b4)
* Create action, you need to use **Apify (3.0.1)** from the list as there lives this PR code. 
![image](https://github.com/apify/apify-zapier-integration/assets/11837643/e8210f8c-05fc-4c95-819e-66fa4e23b6c2)
* Login to your Apify account using token
* Choose Run Actor event
* Select Actor and 🎉  you will see the generated input.

# Screen of input for Web Scraper
![Screenshot 2023-08-08 at 14 28 54](https://github.com/apify/apify-zapier-integration/assets/11837643/bc10ab3c-7ba4-4683-90a3-f2bb0a364c83)
![Screenshot 2023-08-08 at 14 29 04](https://github.com/apify/apify-zapier-integration/assets/11837643/4022f988-74a0-4db2-a218-624ba6ce8f5d)
